src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,52 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(name, health);
+    this.hidden = false;
+
+    if (!Animal.alive) {
+      Animal.alive = [];
```

Comment:
this expression should be used not only in the Herbivore class. All animals alive from start, Carnivore too. So move it to 9 line, and avoid code duplicate from 31-33 lines